build-system/tasks/lint.js

```diff
@@ -161,13 +162,16 @@ function runLinter(stream, options) {
 }
 
 /**
- * Extracts the list of JS files in this PR from the commit log.
+ * Extracts the list of lintable files in this PR from the commit log.
  *
  * @return {!Array<string>}
  */
-function jsFilesChanged() {
+function lintableFilesChanged() {
   return gitDiffNameOnlyMaster().filter(function(file) {
-    return fs.existsSync(file) && path.extname(file) == '.js';
+    return (
+      fs.existsSync(file) &&
+      (path.extname(file) == '.js' || path.basename(file) == 'OWNERS')
+    );
   });
 }
```

Comment:
Shouldn't we be using `lintGlobs` in `build-system/test-configs/config.js`?

Reply:
This was needed in the past when we listed individual files (including negative globs) in `lintGlobs`. Now, we only list two globs in `lintGlobs`: `**/*.js` and `**/OWNERS`. All other negative globs are listed in the global `.eslintignore` (in order to work with IDE plugins). So, when `--files` or `--local_changes` are used, we needn't bother looking at `lintGlobs`.

Reply:
It feels like we're duplicating logic here, though. I get that this function is used only when using these flags, but IMO a cleaner way would be to glob match the files with `lintGlobs`. Or else we'd have to remember to update this function whenever `lintGlobs` is updated.

Reply:
Aha, I now see what you're saying. Agreed. Will do in a follow up.

Reply:
Sent out #24946. Thanks for keeping me honest!

Reply:
:)